packages/holocron-dev-server/src/utils/language-packs.js

```diff
@@ -0,0 +1,150 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import jsonParse from 'json-parse-context';
+
+import { volume, ufs } from './virtual-file-system';
+import { getModulesPath, getLocalesPathForModule } from './paths';
+import { logModuleLanguagePacksLoaded } from './logs';
+import { createLanguagePackWatcher } from './watcher';
+
+export const localePathName = 'locale';
+export const languagePackFileNames = ['copy', 'integration'];
+
+export function readJsonFile(filePath) {
+  const json = ufs.readFileSync(filePath).toString();
+  return jsonParse(json);
+}
+
+export function loadModuleLanguagePack({
+  moduleName, modulePath, localePath = localePathName, locale,
+}) {
+  let languagePackPath = path.join(modulePath, localePath, locale);
+
+  if (!ufs.existsSync(languagePackPath)) {
+    languagePackPath = path.join(modulePath, localePath, `${locale}.json`);
+  }
+
+  if (ufs.existsSync(languagePackPath)) {
+    const fileStats = ufs.statSync(languagePackPath);
+
+    if (fileStats.isDirectory()) {
+      const languagePack = {};
+      const links = {};
+      const scan = [
+        ...languagePackFileNames,
+        locale,
+        moduleName,
+      ];
+      scan
+        .map((fileName) => `${fileName}.json`)
+        .concat('links')
+        .map((fileName) => path.join(languagePackPath, fileName))
+        .filter((filePath) => ufs.existsSync(filePath))
+        .forEach((filePath) => {
+          if (filePath.endsWith('links')) {
+            scan
+              .map((fileName) => path.join(filePath, `${fileName}.json`))
+              .filter((linksFilePath) => ufs.existsSync(linksFilePath))
+              .forEach((linksFilePath) => Object.assign(links, readJsonFile(linksFilePath)));
+          } else {
+            Object.assign(languagePack, readJsonFile(filePath));
+          }
+        });
+      languagePack.links = links;
+      return languagePack;
+    }
+    if (fileStats.isFile()) {
+      return readJsonFile(languagePackPath);
+    }
+  }
+
+  return null;
+}
+
+export function loadModuleLanguagePacks({ moduleName, modulePath, localePath = localePathName }) {
+  const languagePacksPath = path.join(modulePath, localePath);
+  if (ufs.existsSync(languagePacksPath)) {
+    return ufs
+      .readdirSync(languagePacksPath)
+      .map((locale) => locale.replace(/(\..*)$/, '').toLowerCase())
+      .map((locale) => [
+        locale,
+        loadModuleLanguagePack({
+          moduleName, modulePath, localePath, locale,
+        }),
+      ])
+      .filter(([, languagePack]) => !!languagePack);
+  }
+  return [];
```

Comment:
```suggestion
  if (!ufs.existsSync(languagePacksPath)) return [];

  return ufs
    .readdirSync(languagePacksPath)
    .map((locale) => locale.replace(/(\..*)$/, '').toLowerCase())
    .map((locale) => [
      locale,
      loadModuleLanguagePack({
        moduleName, modulePath, localePath, locale,
      }),
    ])
    .filter(([, languagePack]) => !!languagePack);
```

just a style suggestion